### DISCUSSION TITLE
Chart animations

### DISCUSSION THREAD

**Discussion by sixaphone** [mantinedev/mantine#5672] [Feature requests]
I am using mantine/charts in my project and I love their simplicity and design. Is it possible to add the option to allow the underlying recharts animations

**Comment by sixaphone** [comment]
If you like this feature I am open to doing a PR on it

**Comment by sami-azzam** [comment]
+1, I also was trying to achieve entrance animations on Donut Chart, similar to [this one](https://recharts.org/en-US/examples/PieChartWithPaddingAngle).

I didn't find anything in the documentation, is there a workaround to do this in mantine/charts @rtivital ?
and if not, are there plans to support it in the future?

Thanks a lot for your time!

**Reply by sixaphone** [reply]
The default flag is set to false and cant be changed. We just need to expose it and it should would by default
